compiler/noirc_driver/src/lib.rs

```diff
@@ -265,7 +273,7 @@ fn compile_contract_inner(
         }
 
         let function = match compile_no_check(context, options, function_id, None, true) {
-            Ok(function) => function,
+            Ok((function, _warnings)) => function,
```

Comment:
Wouldn't this hide warnings when compiling contracts?

Reply:
Yes, that's why I named it `_warnings`, in case we want to handle it in future, but I can also do it in this PR.

Reply:
I am closing the conversation, the created issue also mention warnings for contracts